README.md

```diff
@@ -133,6 +133,10 @@ docker build -t rac .
 docker run -p 3000:3000 rac
 ```
 
+### Getting the analyst email yourself
+
+It is often useful to quickly see what the analyst email looks like. If the `LDAP_UID` environment variable is not set and the user provides their email address on the contact info screen then the server will send the analyst email unencrypted to the user. This is solely for development purposes. Note that the review apps are configured this way.
+
```

Comment:
A quick question

For example If we aren't using docker image, would be the instructions to generate test email as follow?
- Locate you local .env file in your project root directory f2/
- Ensure that LDAP_UID is not assigned to any value
-  In your terminal run the following commands 
```npm run build``` 
followed by 
```npm run prod``` 
- Start the application in your browser and start fill in form data and ensure that you provide your email in this 
http://localhost:3000/contactinfo page 
 ...etc

- Check your inbox

Reply:
Would the above instructions generate the email successfully?  Or will it be missing a few extra steps?  :)

Reply:
actually, looking at the scripts, `npm run prod` does the build as well as running the server
! Good suggestion to make the instructions more explicit, I'll add ☝️

Reply:
Also, small issue we found this morning with following the above steps, currently in master when using the prod build it throws <404> with report submission. Can we look into this today together? 

Thank you so much @sastels